compiler/tflite2circle/src/BuildBuiltinOptions/ReshapeOptions.cpp

```diff
@@ -25,7 +25,9 @@ flatbuffers::Offset<circle::ReshapeOptions>
 build_circle_ReshapeOptions(flatbuffers::FlatBufferBuilder &fb, const tflite::Operator *op)
 {
   auto tflite_builtin_options = op->builtin_options_as_ReshapeOptions();
-  assert(tflite_builtin_options);
+  if (tflite_builtin_options == nullptr)
+    return 0;
```

Comment:
Is it okay to mismatch with return type? or `return flatbuffers::Offset<void>()`?

Reply:
`0` is casted to `Offset`.
in schema_generated.h file I counld see these codes
```cxx
flatbuffers::Offset<void> details = 0,
...
flatbuffers::Offset<void> array_segments = 0,
...
```